.gitattributes

```diff
@@ -9,6 +9,7 @@
 /.styleci.yml export-ignore
 /.travis.yml export-ignore
 /.github export-ignore
+/mkdocs.yml export-ignore
```

Comment:
Seems like a duplicate now, the same is present on line 15?

Reply:
doh!